backend/src/authConfig.ts

```diff
@@ -0,0 +1,33 @@
+const passportConfig = {
+    credentials: {
+        tenantName: 'stockhubb2c.onmicrosoft.com',
+        clientID: 'dc30ef57-cdc1-4a3e-aac5-9647506a72ef',
```

Comment:
issue(blocking): should comes from env file or github secret.

Reply:
fait sur PR 36